src/components/ChannelList/hooks/usePaginatedChannels.ts

```diff
@@ -120,7 +121,18 @@ export const usePaginatedChannels = <
   };
 
   const loadNextPage = hasNextPage ? queryChannels : undefined;
-  const refreshList = () => queryChannels('refresh');
+  const refreshList = () => {
+    const now = new Date();
+    // Only allow pull-to-refresh 5 seconds after last successful refresh.
+    // @ts-ignore
```

Comment:
what's being ignored here?

Reply:
Date comparison on next line! Any clue whats a nice way to fix it?
<img width="1041" alt="Screenshot 2020-12-07 at 17 31 22" src="https://user-images.githubusercontent.com/11586388/101377470-26219000-38b2-11eb-82f6-805f3bc0bd08.png">

Reply:
oh if you do a `.getTime()` for both `now` and `lastRefresh.current` then it'll be fine. it's saying the left side of your argument isn't returning a number because neither type is a number